cdc/owner.go

```diff
@@ -204,7 +205,14 @@ func (o *Owner) newChangeFeed(
 	if info.Engine == model.SortInFile {
 		err = util.IsDirAndWritable(info.SortDir)
 		if err != nil {
```

Comment:
Could we try to create folders no matter what errors occur, such as no permissions, strange io errors, etc.?

Reply:
addressed